sycl/test/check_device_code/matrix/matrix-nvptx-bf16-test.cpp

```diff
@@ -0,0 +1,279 @@
+// REQUIRES: gpu, cuda
```

Comment:
Is there a reason to require `gpu` here? Even if `cuda` can be set without a GPU, this test is only codegen so it should be fine without the hardware.

Reply:
Yes I think you're right.  I've removed REQUIRES: gpu